src/components/TopBar/CalendarEventConversationActions.vue

```diff
@@ -0,0 +1,118 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { computed, ref } from 'vue'
+import { isNavigationFailure, NavigationFailureType } from 'vue-router'
+import { useRouter, useRoute } from 'vue-router/composables'
+
+import IconArchive from 'vue-material-design-icons/Archive.vue'
+import IconDelete from 'vue-material-design-icons/Delete.vue'
+
+import { showError } from '@nextcloud/dialogs'
+import { t } from '@nextcloud/l10n'
+
+import NcButton from '@nextcloud/vue/components/NcButton'
+import NcDialog from '@nextcloud/vue/components/NcDialog'
+
+import { useStore } from '../../composables/useStore.js'
+import { hasTalkFeature } from '../../services/CapabilitiesManager.ts'
+
+const supportsArchive = hasTalkFeature('local', 'archived-conversations-v2')
+
+const props = defineProps<{
+	token: string,
+	isHighlighted: boolean,
+}>()
+
+const store = useStore()
+const router = useRouter()
+const route = useRoute()
+const showDialog = ref(false)
+
+const isModerator = computed(() => store.getters.isModerator)
+
+/**
+ * Delete conversation
+ */
+async function deleteEventConversation() {
+	try {
+		if (route?.params?.token === props.token) {
+			await router.push({ name: 'root' })
+				.catch((failure) => !isNavigationFailure(failure, NavigationFailureType.duplicated) && Promise.reject(failure))
+		}
+		await store.dispatch('deleteConversationFromServer', { token: props.token })
+	} catch (error) {
+		console.error(`Error while deleting conversation ${error}`)
+		showError(t('spreed', 'Error while deleting conversation'))
+	}
+}
+
+/**
+ * Archive conversation
+ */
+async function archiveEventConversation() {
+	await store.dispatch('toggleArchive', { token: props.token, isArchived: false })
+}
+</script>
+
+<template>
+	<div class="conversation-actions"
+		:class="{ 'conversation-actions--highlighted': props.isHighlighted }">
+		<p>{{ t('spreed', 'Meeting conversations are archived after 7 days of no activity.') }}</p>
```

Comment:
```suggestion
		<p>{{ t('spreed', 'Meeting conversations are archived after {amount} days of no activity.') }}</p>
```

Reply:
You mean plurals:
```suggestion
		<p>{{ n('spreed', 'Meeting conversations are archived after %n day of no activity.', 'Meeting conversations are archived after %n days of no activity.', expirationDays) }}</p>
```

Reply:
with [Intl.DurationFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DurationFormat) we can even show "4 weeks", so I'd put the whole time definition into param